rpm/mic.spec

```
@@ -78,6 +78,7 @@ Requires:   yum >= 3.2.24
 %setup -q -n %{name}-%{version}
 
 %build
+echo %{version} > VERSION
```

Depending on what is the use of the result, this might work better WRT test builds from revisions without tag:

```suggestion
sed 's/+.*//' <<<"%{version}" > VERSION
```

---

Thanks, will fix.